usaspending_api/disaster/tests/fixtures/helpers.py

```diff
@@ -64,16 +64,11 @@ def post_for_amount_endpoint(client, url, def_codes, award_type_codes=None, awar
 
     @staticmethod
     def patch_datetime_now(monkeypatch, year, month, day):
-        patched_datetime = datetime.datetime(year, month, day, tzinfo=datetime.timezone.utc)
+        def patched_datetime():
+            return datetime.datetime(year, month, day, tzinfo=datetime.timezone.utc)
 
-        class PatchedDatetime(datetime.datetime):
-            @classmethod
-            def now(cls, *args):
-                return patched_datetime
-
-        monkeypatch.setattr("usaspending_api.submissions.helpers.datetime", PatchedDatetime)
-        monkeypatch.setattr("usaspending_api.disaster.v2.views.disaster_base.datetime", PatchedDatetime)
-        usaspending_api.common.helpers.fiscal_year_helpers.current_fiscal_year = lambda: year
```

Comment:
There's a slight chance that removing this fixed our sporadic test failures.  It was just wholesale replacing the function instead of monkeypatching it.

Reply:
😱 🙀 

🤞